one.css

```diff
@@ -417,20 +432,145 @@ Modal
     padding: 0.5rem;
 }
 
+
+
 /*
-Grids
+Navigation Bar
 */
+.nav {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 100%;
+    padding: 1.5rem;
+    background-color: var(--orange);
+}
 
-/* Grid with 2 columns */
-.grid-col-2 {
-    display: grid;
-    grid-template-columns: 1fr 1fr;
+.navbar-brand {
+    color: black;
+    font-size: 1.5rem;
+    font-weight: 700;
+    margin: 0;
+    padding: 0;
+    text-decoration: none;
 }
 
-/* Grid with 3 colums */
-.grid-col-3 {
-    display: grid;
-    grid-template-columns: 1fr 1fr 1fr;
+.navbar-nav {
+    padding: 0;
+    margin: 0;
+}
```

Comment:
Can we do the follwowing?
```
*{
margin: 0;
padding:0;
}
```
Please let me know if this doesn't work..

Reply:
We can do the following you mentioned and its working. Its removing padding and margin on every element of the page which I don't need.What I'm trying to do is removing the padding and margin on the logo of the website and the list that contains the items on the navigation bar.